README.md

```diff
@@ -12,10 +12,11 @@ cd motohelper/
 gem install bundler
 bundle
 rake db:setup
+rake db:migrate
```

Comment:
Is this necessary? ``` rake db:setup ``` create db, load data schema and execute seeds. In really big projects rake db:migrate may not working in this situation.